### PR TITLE
added mime types

### PR DESCRIPTION
**Pull request by jacksonargo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ CMD ["make", "test"]
 
 FROM gcr.io/distroless/base-debian10 as vvgo
 COPY --from=builder vvgo vvgo
+COPY ./etc /etc
 COPY ./public /public
```
Comment (0az) [inline Dockerfile:21]:
```suggestion
COPY ./data/mime.types /etc/
COPY ./public /public
COPY --from=builder vvgo vvgo
```
We should probably keep all of these assorted files to the `data` tree.

This also reorders the `COPY`s for better caching.

Comment (jacksonargo) [reply Dockerfile:21]: Right now, everything in `./data` directory gets globbed up and built into the binary, so I don't want to put it in `./data`

**Comment by 0az** [comment]
Not 100% about the gitignore change, though.

**Comment by jacksonargo** [comment]
Will this show up?

**Pull request merged**